Modules/BenchPress.Azure/Public/Confirm-ApiManagementDiagnostic.ps1

```diff
@@ -0,0 +1,60 @@
+# INLINE_SKIP
+using module ./../Classes/ConfirmResult.psm1
+
+. $PSScriptRoot/../Private/Connect-Account.ps1
+# end INLINE_SKIP
+
+function Confirm-ApiManagementDiagnostic {
+  <#
+    .SYNOPSIS
+      Confirms that an API Management Diagnostic exists.
+
+    .DESCRIPTION
+      The Confirm-AzBPApiManagementDiagnostic cmdlet gets an API Management Diagnostic using the specified API
+      Diagnostic, API, API Management Service, and Resource Group names.
+
+    .PARAMETER ResourceGroupName
+      Specifies the name of the resource group under which an API Management service is deployed.
+
+    .PARAMETER ServiceName
+      Specifies the name of the deployed API Management service.
+
+    .PARAMETER ApiId
```

Comment:
Doc parameter list and actual parameter list do not match. Actual parameter list is missing ApiId

Reply:
Good catch. Fixed.